Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -43,6 +42,39 @@ def __init__(
         self.only_escalated = only_escalated
         self.auth_token = ""
 
+    def raw_api_request(
```

Comment:
What was the need for a new method?
Why not use the existing api_request method and add the resp_type argument and change the return type annotation accordingly?

Reply:
I did that, but I had issues with mypy on the pre-commit hook
which is why I separated it

Reply:
OK, I understand the problem